README.md

```diff
@@ -1,5 +1,4 @@
-# Slack plugin for Jenkins
-
+# Slack plugin for Jenkins [![Build Status](https://travis-ci.org/jenkinsci/slack-plugin.svg?branch=master)](https://travis-ci.org/jenkinsci/slack-plugin)
```

Comment:
There's no Travis CI build so no need to provide a link in the README.  Can you amend this commit?